src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/FormatControl.CurrencyFormatType.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System.Windows.Forms.Design;
+
+internal partial class FormatControl
+{
+    private class CurrencyFormatType : FormatTypeClass
+    {
+        private readonly FormatControl _owner;
+
+        public CurrencyFormatType(FormatControl owner)
+        {
+            _owner = owner;
+        }
+
+        public override string TopLabelString
+        {
+            get
+            {
+                return SR.BindingFormattingDialogFormatTypeCurrencyExplanation;
+            }
+        }
```

Comment:
can be simplified others in this statement.

```suggestion
        public override string TopLabelString => SR.BindingFormattingDialogFormatTypeCurrencyExplanation;
```